local-volume/provisioner/deployment/kubernetes/helm/provisioner/values.yaml

```diff
@@ -0,0 +1,21 @@
+common:
+  namespace: default
+  defaultHostDir: /mnt/disks
+  defaultMountDir: /local-disks
+configmap:
```

Comment:
I think we should not have any default configmap

Reply:
Oh I misunderstood.  This is not quite a default values file, this is where users will specify their config and the configmap will be generated from it.  I guess this is fine then.  Does it have to be called values.yaml, or can we specify other yaml files?

Reply:
Correct, values.yaml that is where we can program defaults and user provides their values. If you do not want any defaults there, then I can move all defaults into the template. Please let me know when you think.

Reply:
I think it is fine to keep it here, to provide an example to users on how to fill in their values.  When you call "helm template", can you pass in a different values.yaml file?

Reply:
You can specify another values.yaml by using  --values keyword in the command line. File provided in --values will be merged with values in the chart folder, values specified in command line take precedence over values in the folder.